pkg/kustomize/kustomize.go

```diff
@@ -110,6 +113,37 @@ func execAdd(fsys filesys.FileSystem, path string, AllTypesFiles map[fileType][]
 	return nil
 }
 
+func remove(s []string, i int) []string {
```

Comment:
This function remove item changing the slice order. It is correct?

Reply:
correct, the order does not matter in this list

Reply:
Ok, perfect